src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs

```diff
@@ -117,7 +117,7 @@ public static bool IsServerDomainJoined()
         // Returns 'true' if the client is running on a Windows OS
         public static bool IsWindows()
         {
-            return RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+            return Platform.MatchesCurrentPlatform(PlatformID.Windows);
```

Comment:
Now that I've centralized other Platform detection into the Platform class, I wanted to be consistent in using it.  Nothing required this change other than code consistency, and I can revert if you prefer.